business_requirement_deliverable_categ/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================================
+Business Requirement Deliverable Task Category
+==============================================
+
+This module extends module business_requirement_deliverable and business_requirement_project.
+
+In business requirement if create a new business requirement resource line and if it is for a "Task" resource you could add the task category.
```

Comment:
In business requirement, when creating a new resource line for a "Task" resource, you can add the task category.